src/screens/APIUtils/APIUtils.res

```diff
@@ -210,111 +220,188 @@ let useGetURL = () => {
         `analytics/v1/connector_event_logs?type=Payment&payment_id=${payment_id}`
       | None => ""
       }
+
+    /* SAMPLE DATA */
+    | GENERATE_SAMPLE_DATA => `user/sample_data`
+
+    /* VERIFY APPLE PAY */
+    | VERIFY_APPLE_PAY => `verify/apple_pay`
+
+    /* PAYPAL ONBOARDING */
+    | PAYPAL_ONBOARDING => `connector_onboarding`
+
+    /* BUSINESS PROFILE */
+    | BUSINESS_PROFILE =>
+      switch id {
+      | Some(id) => `account/${merchantId}/business_profile/${id}`
+      | None => `account/${merchantId}/business_profile`
+      }
+
+    /* API KEYS */
+    | API_KEYS =>
+      switch methodType {
+      | Get => `api_keys/${merchantId}/list`
+      | Post =>
+        switch id {
+        | Some(key_id) => `api_keys/${merchantId}/${key_id}`
+        | None => `api_keys/${merchantId}`
+        }
+      | Delete => `api_keys/${merchantId}/${id->Option.getOr("")}`
+      | _ => ""
+      }
+
+    /* DISPUTES EVIDENCE */
+    | ACCEPT_DISPUTE =>
+      switch id {
+      | Some(id) => `disputes/accept/${id}`
+      | None => `disputes`
+      }
+    | DISPUTES_ATTACH_EVIDENCE =>
+      switch id {
+      | Some(id) => `disputes/evidence/${id}`
+      | _ => `disputes/evidence`
+      }
+
+    /* PMTS COUNTRY-CURRENCY DETAILS */
+    | PAYMENT_METHOD_CONFIG => `payment_methods/filter`
+
+    /* USER MANAGEMENT */
+    | USER_MANAGEMENT => {
+        let userUrl = `user`
+        switch userRoleTypes {
+        | USER_LIST => `${userUrl}/user/list`
+        | ROLE_LIST => `${userUrl}/role/list`
+        | ROLE_ID =>
+          switch id {
+          | Some(key_id) => `${userUrl}/role/${key_id}`
+          | None => ""
+          }
+        | NONE => ""
+        }
+      }
+
+    /* USERS */
     | USERS =>
       let userUrl = `user`
+
       switch userType {
-      | #NONE => ""
-      | #USER_DATA => `${userUrl}/data`
-      | #MERCHANT_DATA => `${userUrl}/data`
-      | #RESEND_INVITE
-      | #INVITE_MULTIPLE =>
-        switch queryParamerters {
-        | Some(params) => `${userUrl}/user/${(userType :> string)->String.toLowerCase}?${params}`
-        | None => `${userUrl}/user/${(userType :> string)->String.toLowerCase}`
-        }
+      // DASHBOARD LOGIN / SIGNUP
```

Comment:
Comments are not unified - 

One place - /* PMTS COUNTRY-CURRENCY DETAILS */
Other place -// DASHBOARD LOGIN / SIGNUP

Reply:
`// DASHBOARD LOGIN / SIGNUP` 
This one is a sub module under the main /* USERS */ module . We have decided to follow the convention as 
main modules - ` /* PMTS COUNTRY-CURRENCY DETAILS */`
and sub modules - `// DASHBOARD LOGIN / SIGNUP`